packages/lpa-questionnaire-web-app/src/validators/development-plan.js

```diff
@@ -0,0 +1,27 @@
+const { body } = require('express-validator');
+
+const rules = () => {
+  const developmentPlanRef = 'has-plan-submitted';
+  const developmentPlanValues = ['yes', 'no'];
+  const developmentPlanTextRef = 'plan-changes-text';
+
+  return [
+    body(developmentPlanRef)
+      .notEmpty()
+      .withMessage(
+        'Has any Development Plan Document or Neighbourhood Plan relevant to this appeal been submitted for examination (or in the case of a Neighbourhood Plan, been examined and is awaiting a referendum)?'
+      )
+      .bail()
+      .isIn(developmentPlanValues),
+    body(developmentPlanTextRef)
+      .if(body(developmentPlanRef).equals('yes'))
+      .notEmpty()
+      .withMessage(
+        'Tell us about any substantive changes in the progress of the emerging plan, and their relevance to this appeal (if the plan may not be adopted before the Inspector issues their decision).'
```

Comment:
This is not the correct error. Should be 'Enter the relevant information about the plans and this appeal'. Need to check these from the ticket.

Reply:
Corrected